src/js/disability-benefits/components/ClaimEstimate.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import moment from 'moment';
+
+export default function ClaimEstimate({ maxDate }) {
+  const estimatedDate = moment(maxDate);
+  const today = moment().startOf('day');
+
+  if (!maxDate || !estimatedDate.isValid() || estimatedDate.isAfter(moment(today).add(2, 'years'))) {
```

Comment:
Do you need to check `!maxDate` here? You've already wrapped `maxDate` in moment, so it seems like any validity checks should use the wrapped variable, or that `maxDate` shouldn't be wrapped if it is already invalid